_data/pubs.yml

```diff
@@ -9,6 +9,33 @@
   month: April
   tags: [hci]
   type: full paper
+  
+- title: "For God’s sake! Considering Religious Beliefs in HCI Research: A Case of Islamic HCI"
+  authors: Samia Ibtasam
+  year: 2021 
+  venue: Proceedings of the 2021 CHI Conference on Human Factors in Computing Systems (CHI ’21)
+  url: "https://samiaibtasam.com/wp-content/uploads/2021/05/A-Case-of-Islamic-HCI.pdf"
+  month: April
+  tags: [hci]
+  type: alt.chi paper
+  
+- title: "From the Margins to the Centre: Defining New Mission and Vision for HCI Researchers in South Asia"
+  authors: Pranjal Jain, Samia Ibtasam, Sumita Sharma, Nilavra Bhattacharya, Anupriya Tuli, Dilrukshi Gamage, Dhruv Jain, Rucha Tulaskar, Priyank Chandra, Lubna Razaq, Rahat Jahangir Rony, Deepak Ranjan Padhi, Mohit Jain, Suleman Shahid, Nova Ahmed, Devanuj Kanta Balkrishan, Pushpendra Singh
+  year: 2021 
+  venue: Proceedings of the 2021 CHI Conference on Human Factors in Computing Systems (CHI ’21)
+  url: "https://samiaibtasam.com/wp-content/uploads/2021/05/southasiaHCI.pdf"
+  month: April
+  tags: [hci]
+  type: full paper
+  
+- title: "'We Even Borrowed Money From Our Neighbor': Understanding Mobile-based Frauds Through Victims' Experiences"
+  authors: Authors: Lubna Razaq, Tallal Ahmad, Samia Ibtasam, Umer Ramzan, Shrirang Mare
```

Comment:
This needs a minor fix, Removing the second `Authors` should prepare this for a merge.